plugin/web/extensions/sidebar/modals/datacatalog/components/content/DatasetsPage/DatasetTree/FileTree.tsx

```diff
@@ -30,6 +30,17 @@ const FileTree: React.FC<Props> = ({
     select(dataID);
   }, []);
 
+  useEffect(() => {
+    const { expandedKeys } = window as any;
+    if (expandedKeys) {
+      const newExpandedKeys = [...expandedKeys];
+      setExpandedKeys(newExpandedKeys);
+    }
+    setTimeout(() => {
+      (window as any).expandedKeys = undefined;
+    }, 500);
```

Comment:
A couple questions:
- Why does this useEffect's deps array have two functions not used in the useEffect?
- We discussed on slack, but you shouldn't be able to access the window object from a plugin, so this expandedKeys should not be accessible (if there is anything to discuss around this, please do on slack)

Reply:
Removed the two functions that are not used in the useEffect

Reply:
Done!